app/assets/javascripts/components/Register.jsx

```diff
@@ -0,0 +1,107 @@
+class Register extends React.Component {
+
+  constructor(props) {
+    super();
+    this.state = {"success": null};
```

Comment:
you dont need quotes around javascript string object keys:
e.i. `{success: null}`